app/views/shared/_favicons.html.erb

```diff
@@ -0,0 +1,20 @@
+<link rel="apple-touch-icon-precomposed" sizes="57x57" href="/blog/apple-touch-icon-57x57.png" />
```

Comment:
Might be good with these to look for `ENV['RAILS_RELATIVE_URL_ROOT']` so we don't have a hard coded dependency to where we're serving it.

Something like:

```
<link rel="apple-touch-icon-precomposed" sizes="57x57" href="<%= ENV['RAILS_RELATIVE_URL_ROOT'] %>/apple-touch-icon-57x57.png" />
```